src/lib/components/UrlChooser/index.js

```diff
@@ -50,6 +53,7 @@ class UrlChooser extends BaseElement {
             pattern="https?://.+"
             minlength="7"
             @keyup="${this.onUrlKeyup}"
+            value="${urlSearchParam}"
```

Comment:
I'm pretty sure that the `html` template tag handles all the necessary escaping here, and that it's safe to pass in user-defined input. If that's not the case, please let me know.

Reply:
I think it would be better to pass it as a param into the component. But since this component is not reused atm, let's keep it as is.